packages/contract/src/contract/index.ts

```diff
@@ -11,7 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-import '@polkadot/api-augment'
+// import '@polkadot/api-augment'
```

Comment:
Why commented out?

Reply:
Because it only needs to be imported once and it's imported in `packages/contract/src/index.ts`. I was experimenting with removing it but it breaks the TS build. I've deleted the commented one.